portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsUploadField.testcase

```diff
@@ -451,6 +451,259 @@ definition {
 			successPageTitle = "Thank you.");
 	}
 
+	@description = "LPS-84340 - Verify that a Form can be submitted with the Upload Field"
+	@priority = "5"
+	test CanBeSubmitted {
+		property portal.acceptance = "true";
+		property portal.upstream = "true";
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName();
+
+		Form.editDescription();
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Upload");
+
+		var fieldUploadField = FormFields.getFieldName();
+
+		Form.editFieldLabelMetal(fieldValue = "Upload Field");
+
+		Form.closeSidebar();
+
+		Form.save();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoEditFormByName();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormPortlet.waitFormRenderer(text = "Submit");
+
+		FormFields.viewUploadField(
+			fieldLabel = "Upload Field",
+			fieldName = "${fieldUploadField}");
+
+		ItemSelector.gotoItemSelectorViaSelectButton();
+
+		ItemSelector.uploadFile(
+			navTab = "Documents and Media",
+			uploadFileName = "Document_1.jpg");
+
+		FormPortlet.submitSuccessfully();
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoViewEntries();
+
+		FormsAdminNavigator.gotoView();
+
+		FormsAdmin.viewUploadFile(
+			fieldLabel = "Upload Field",
+			fieldName = "${fieldUploadField}",
+			uploadFileName = "Document_1.jpg");
+	}
+
+	@description = "LPS-125462 - Verify that a new user can submit a Form after uploading a file"
+	@priority = "5"
+	test CanBeSubmittedByANewUser {
+		property portal.acceptance = "true";
+		property portal.upstream = "true";
+
+		JSONUser.addUser(
+			userEmailAddress = "user1@liferay.com",
+			userFirstName = "user1",
+			userLastName = "user1",
+			userScreenName = "user1");
+
+		JSONUser.setFirstPassword(
+			agreeToTermsAndAnswerReminderQuery = "true",
+			requireReset = "false",
+			userEmailAddress = "user1@liferay.com");
+
+		JSONUser.addUserToSite(
+			groupName = "Guest",
```

Comment:
@leticiachiu, why did you decide to use `Guest` instead of `Liferay`, which is the default site?

Actually, what is the meaning of `groupName="Guest"`? Is it making the test consider the created user as a Guest user? If so, I understand that we should change it once we already have another Poshi test verifying if a Guest user can upload a file.

Reply:
@natocesarrego The variable would be `Liferay` if I had created `JSONGroup.addGroup(groupName = "Liferay");` before, but I don't think is necessary in this test.. for using this macro, I can only add the new user as a guest. The other test with guest user is different because is testing if a form can be submitted without any account, but this one we're creating a new user.

Reply:
@leticiachiu, I believe that use the parameter `groupName=Liferay` probably won't be a problem, because the site named `Liferay` is created every time the portal is initialized, actually, this is the default site of portal.

My only concern to use the parameter  `groupName=Guest` is the issue reported by you in https://issues.liferay.com/browse/LPS-136679 which is not fixed yet. If we use `groupName=Guest`, will the test `CanBeSubmittedByANewUser` pass?

Maybe, we should have a test for each scenario, I mean using both parameters, `groupName=Liferay` and  `groupName=Guest`. What do you think? Let's discuss about it after our daily meeting today. :wink:

Reply:
![image](https://user-images.githubusercontent.com/69634035/128542942-de28c17d-a10b-4493-bd8c-6d7f647e1e57.png)
It failed when I tested with groupName=Liferay